src/js/serial_backend.js

```diff
@@ -148,9 +149,10 @@ export function initializeSerialBackend() {
 
                     function onFinishCallback() {
                         finishClose(toggleStatus);
+                        GUI.updateManualPortVisibility();
                     }
 
-                    mspHelper.setArmingEnabled(true, false, onFinishCallback);
+                    mspHelper?.setArmingEnabled(true, false, onFinishCallback);
```

Comment:
Race condition?

Reply:
Indeed - the steps to reproduce it are:

1. Start a new instance of the configurator (it must not have been connected to an FC yet as otherwise `mspHelper` will be defined)
    - I suppose the same can be reproduced with a slow/unresponsive FC
3. Connect to a TCP address that won't respond (or will be very slow in doing so) - e.g. in the context of these changes - `10.0.0.2` when in the WiFi network of an ELRS RX
4. Click connect and then subsequently click disconnect

Here is a small demo:

![setArmingEnabled-race-condition](https://github.com/betaflight/betaflight-configurator/assets/3475899/897b2c7d-3e2e-40a2-aa24-1780f8b537a5)